v1/src/main/java/com/google/cloud/teleport/spanner/ddl/InformationSchemaScanner.java

```diff
@@ -156,18 +162,32 @@ Statement databaseOptionsSQL() {
         return Statement.of(
             "SELECT t.option_name, t.option_type, t.option_value "
                 + " FROM information_schema.database_options AS t "
-                + " WHERE t.catalog_name = '' AND t.schema_name = ''");
+                + " WHERE t.schema_name = ''");
```

Comment:
What is the impact of this? The databaseOptions are always have schema_name = '' and catalog_name can be anything?

Reply:
Right, only the default schema (in pg it is public, and in googlesql it is empty:"") could have database Options.

The catalog_name = '' is not that useful, because it is always empty. And unlikely we will use it.

Reply:
catalog_name always just one value, for GoogleSQL it is empty, and for Pg it is the database name. 
So we can ignore catalog_name at here and also all other places.